app/lib/actions/katello/agent_action.rb

```diff
@@ -28,6 +28,7 @@ def plan(host, options)
       end
 
       def run(event = nil)
+        return if event == Dynflow::Action::Skip
```

Comment:
can you add an additional item in the below case statement with a blank body to achieve the same? Would like a unit test as well :)

Reply:
We decided this is not possible after discussion in IRC, marking resolved.